middleware/http/source/outbound/handle.cpp

```diff
@@ -97,17 +97,17 @@ namespace casual
                         auto reply = common::message::reverse::type( message, process::handle());
                         reply.domain = state.identity;
 
-                        reply.content.services = algorithm::accumulate( message.content.services, decltype( reply.content.services){}, [&state]( auto result, auto& name)
+                        using Services = std::remove_reference_t< decltype( reply.content.services())>;
```

Comment:
I usually use `std::decay_t< ...>` or `traits::remove_cvref_t` (in c++20  `std::remove_cvref_t`).
But perhaps `std::remove_reference_t<...>` is more precise.